web/src/containers/BudgetSummary.js

```diff
@@ -15,22 +15,20 @@ const categoryLookup = {
 function DataRow({ data, title }) {
   return (
     <tr
-      className={ title===categoryLookup.combined ? 'budget-table--subtotal budget-table--row__highlight' : '' }
+      className={
+        title === categoryLookup.combined
+          ? 'budget-table--subtotal budget-table--row__highlight'
+          : ''
+      }
     >
       <th scope="row">{title}</th>
-      <td
-        className="budget-table--number"
-      >
```

Comment:
I love how prettier just randomly switches between these two...

Reply:
It's like a treasure hunt! What code actually changed? You'll just have to keep searching...